posts/enterprise-lacking-cloud-skills.md

```diff
@@ -0,0 +1,63 @@
+# Enterprise lacking skills required for cloud migration
+
+> 66% of I.T. leaders say their organisations are not adequately prepared for the disruptions to come
+
+http://www.gartner.com/newsroom/id/3375817
+
+For all the excitement and talk of cloud technologies it seems the reality is a little less glamorous. With more and more executives backing the move to the cloud, more and more organisations are booting up large migration programmes. The problem is, organisations do not necessarily have the skills to migrate. 66% of I.T. leaders agree in a Gartner report that their organisations are not adequately prepared for this change.
+
+> 81% of enterprises rely on service providers for cloud
+
+https://www.veritas.com/news-releases/2016-06-28-state-of-the-hybrid-cloud-research
+
+Not only that but enterprises are reporting the need to outsource this to service providers, often locking themselves into projects lasting years and ultimately huge bills. 81% of enterprises according to a Veritas survey are relying on service providers to help them migrate.
```

Comment:
Would change the last sentence to one of these:
- "81% of enterprises, according to a Veritas survey, are..."
- "According to a Veritas survey, 81% of enterprises are..."